recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "v0.5.1" %}
+{% set version = "v0.5.2" %}
 
 package:
   name: nbgrader
   version: {{ version }}
 
 source:
   url: https://github.com/jupyter/nbgrader/archive/{{ version }}.tar.gz
```

Comment:
Yeah, I realize that would be the fix but I'm a bit worried that if you have installed `v0.5.1` and then do `conda install nbgrader` it will think that `v0.5.1` is newer than `0.5.2` and hence won't install the updated version:

```
>>> "v0.5.1" > "0.5.2"
True
```

Do you know if this is going to be a problem?